vendor/assets/swagger-ui-dist/absolute-path.js

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
This lib seems to go to a lot of trouble to expose this `getAbsoluteFSPath` method but we're not calling it anywhere. I assume that that's just because we don't need it, because its static files are all in a place where Rails knows how to serve them?

Reply:
This was for backwards compatibility with some really old swagger implementations. We haven't used this ever and don't care.